dist/images/Dockerfile.base

```diff
@@ -48,7 +48,9 @@ RUN cd /usr/src/ && git clone -b branch-22.12 --depth=1 https://github.com/ovn-o
     # lflow: do not send direct traffic between lports to conntrack
     curl -s https://github.com/kubeovn/ovn/commit/54cbe0d1ba2051e640dd3e53498f373362547691.patch | git apply && \
     # northd: add nb option version_compatibility
-    curl -s https://github.com/kubeovn/ovn/commit/06f5a7c684a6030036e2663eecf934b37c3e666e.patch | git apply
+    curl -s https://github.com/kubeovn/ovn/commit/06f5a7c684a6030036e2663eecf934b37c3e666e.patch | git apply && \
+    # northd: skip conntrack when access node local dns ip
+    curl -s https://github.com/kubeovn/ovn/commit/1a4306ab08667346b56703a28f3808c5537fdc14.patch | git apply
```

Comment:
```diff
+        if (strlen(node_local_dns_ip_v4) != 0) {
```

`node_local_dns_ip_v4` may be uninitailized before using.

Reply:
There should be no need to check initialization, because this node_local_dns_ip_v4 is statically initialized 

Static character node_local_dns_ip_v4[INET_ADDRSTRLEN];

Reply:
Before parsing the nb option, we should set node_local_dns_ip_v4:
```diff
+    const char *local_dns_ip = smap_get(&nb->options, "node_local_dns_ip");
+    node_local_dns_ip_v4[0] = 0;
+    if (local_dns_ip) {
```

Reply:
ok， I use memset